src/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {WebView} from 'react-native-webview';
 import {Linking, Platform, SafeAreaView, StyleSheet} from 'react-native';
 
-const version = '2.1.0';
+const version = '2.2.0';
```

Comment:
curious, can we import this from `package.json`?

Reply:
We tried but ran into serious issues and there were security reasons mentioned not to do that either when I looked into it.

Reply:
got it!

Reply:
may be good to have a unit test testing that this version matches package.json's

Reply:
True - I'll add that to my backlog

Reply:
yah, not super urgent but good insurance!